install_config/configuring_gce.adoc

```diff
@@ -11,22 +11,59 @@
 toc::[]
 
 == Overview
-{product-title} can be configured to access an
-link:https://cloud.google.com/compute/docs/disks/[GCE infrastructure], including
+{product-title} can be configured to access a
+link:https://cloud.google.com/compute/docs/disks/[Google Compute Engine (GCE) infrastructure], including
 xref:../install_config/persistent_storage/persistent_storage_gce.adoc#install-config-persistent-storage-persistent-storage-gce[using GCE
 volumes as persistent storage] for application data. After GCE is configured
 properly, some additional configurations will need to be completed on the
 {product-title} hosts.
 
+
 [[gce-configuring-masters]]
 == Configuring Masters
 
-Edit or
+You can set the GCE configuration on your {product-title} master hosts in two ways:
+
+- xref:gce-configuring-masters-ansible[Using Ansible and the advanced installation tool.]
+- xref:gce-configuring-masters-manual[Manually by modifying the *_master-config.yaml_* file.]
+
+
+[[gce-configuring-masters-ansible]]
+=== Configuring {product-title} Masters for GCE with Ansible
+
+During xref:../install_config/install/advanced_install.adoc#install-config-install-advanced-install[advanced installations], GCE can be configured using the `openshift_cloudprovider_kind` parameter, which is configurable in the inventory file.
+
+.Example GCE Configuration with Ansible
+====
+----
+# Cloud Provider Configuration
+#
+  openshift_cloudprovider_kind=gce
+----
+====
+
+
+[NOTE]
+====
+When Ansible configures GCE, the following files are created for you:
+
+- */etc/origin/cloudprovider/gce.conf*
+- *_/etc/origin/master/master-config.yaml_*
+- *_/etc/origin/node/node-config.yaml_*
+====
+
+The advanced installation configures multizone support by default.  If you want single-zone support, edit the */etc/origin/cloudprovider/gce.conf* as shown in xref:configuring-gce-multizone[Configuring Multizone Support in a GCE Deployment].
+
+[[gce-configuring-masters-manual]]
+=== Manually Configuring {product-title} Masters for GCE 
+
+To configure the {product-title} masters for GCE: 
+
+. Edit or
 xref:../install_config/master_node_configuration.adoc#creating-new-configuration-files[create] the
-master configuration file on all masters
-(*_/etc/origin/master/master-config.yaml_* by default) and update the
+master configuration file (*_/etc/origin/master/master-config.yaml_* by default) on all masters and update the
 contents of the `*apiServerArguments*` and `*controllerArguments*` sections:
-
++
 ====
 [source,yaml]
 ----
```

Comment:
Thanks for all the comments. The === around code blocks was in there already. Will keep in mind to edit these out in the future.